numpy/core/tests/test_api.py

```diff
@@ -144,7 +144,7 @@ def test_array_array():
 
 @pytest.mark.parametrize("array", [True, False])
 def test_array_impossible_casts(array):
-    # All builtin types can forst cast as least theoretically
+    # All builtin types can forst cast at least theoretically
```

Comment:
This still needs work. I think it is supposed to be
```
# All builtin types can be force cast,  at least theoretically,
```
@seberg Thoughts?

Reply:
Yes, the commas are indeed missing, thanks!

Reply:
Ah thank you. I had been unable to find an alternative that makes sense - I had tried "_first cast_" but it didn't make sense. Perhaps "_can be forcibly cast_"?

Reply:
oh, hadn't noticed the "forst" :)

Reply:
Fixed, thank you.